cocos/2d/CCSprite.cpp

```diff
@@ -287,9 +287,6 @@ bool Sprite::initWithTexture(Texture2D *texture, const Rect& rect, bool rotated)
         _quad.br.colors = Color4B::WHITE;
         _quad.tl.colors = Color4B::WHITE;
         _quad.tr.colors = Color4B::WHITE;
-        
```

Comment:
Since GLProgramState will be reset by `setTexture` function call, so I remove the comment here.